tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/builtinoperations/FreezeAndIsFrozenTest.java

```diff
@@ -364,6 +371,11 @@ public void testValidComplexMapFreeze() {
                 "was encountered");
     }
 
+    @Test(description = "test recursive tuple")
```

Comment:
IMO, the test name explains what we're testing for and the description doesn't add value here.